iib/web/models.py

```diff
@@ -1296,6 +1360,7 @@ def to_json(self, verbose=True):
             self.target_index_resolved, 'pull_specification', None
         )
         rv['distribution_scope'] = self.distribution_scope
+        rv['build_tags'] = [t.name for t in self.build_tags]
```

Comment:
Interestingly, you didn't add this line to the classes above RequestAdd and RequestRm. What about to add it to the Request class and this line will ne inherited for RequestAdd, RequestRm, RequestRegenerateBundle and RequestCreateEmptyImage.

Reply:
I added it here only because of the inheritance you mentioned.